spec/javascripts/govuk-component/govspeak-enhance-youtube-video-links-spec.js

```diff
@@ -24,6 +24,16 @@ describe('Links to YouTube in govspeak content', function () {
       $('.media-player').remove();
     });
 
+    it("has a protocol scheme of https: or http:", function () {
```

Comment:
This is a bad test – you're not testing the code within `govspeak-enhance-youtube-video-links.js`, instead the code being tested is completely within the test.

I'd expect something to mock `document.location.protocol` and test the outcome, ie the YouTube URL output, though I know this is a pain.